### PR TITLE
docs: add deployed contract docs

### PR DESCRIPTION
**Pull request by MoongateIntern**

### CODE REVIEW

**Comment by MoongateIntern** [comment]
For event tickets:
image

For membership tiers:
image

File: creating-campaign/membership/creatememtier.mdx [hunk 1]
```diff
@@ -81,4 +81,9 @@ title: 'Creating Membership Tiers'
         Once you have finalized your membership program and tiering details, click "Publish" to launch your event.
         <img src="/images/creating-campaign/membership/creating tier/Publish Button.png"/>
     </Step>
+    <Step title="Preview Deployed Contract">
+        After publishing your events, you can preview the deployed contract in the **Advanced Settings**.
```
Comment (prabhanshjn) [inline creating-campaign/membership/creatememtier.mdx]: Please change the following:
1. Events -> Membership
2. deployed contract -> deployed contract address

File: creating-campaign/membership/creatememtier.mdx [hunk 2]
```diff
@@ -81,4 +81,9 @@ title: 'Creating Membership Tiers'
...
+        <Warning>Deployed contract is only visible to published events, not to draft events.</Warning>
```
Comment (prabhanshjn) [inline creating-campaign/membership/creatememtier.mdx]: 1. events -> memberships
2. state that this field cannot be changed

File: creating-campaign/ticketing/createtickets.mdx [hunk 3]
```diff
@@ -121,4 +121,9 @@ title: 'Creating Tickets'
     Once you have finalized your event and ticket details, click "Publish" to launch your event.
     <img src="/images/creating-campaign/ticketing/creating tickets/Publish Button Location.png"/>
   </Step>
+  <Step title="Preview Deployed Contract">
+    After publishing your events, you can preview the deployed contract in the **Advanced Settings**.
```
Comment (prabhanshjn) [inline creating-campaign/ticketing/createtickets.mdx]: 1. events -> event
2. deployed contract -> deployed contract address

File: creating-campaign/ticketing/createtickets.mdx [hunk 4]
```diff
@@ -121,4 +121,9 @@ title: 'Creating Tickets'
...
+    <Warning>Deployed contract is only visible to published events, not to draft events.</Warning>
```
Comment (prabhanshjn) [inline creating-campaign/ticketing/createtickets.mdx]: State that it cannot be changed

**Review by prabhanshjn — CHANGES_REQUESTED** [review]

File: images/creating-campaign/membership/creating tier/deployed contract.png
Comment (prabhanshjn) [inline images/creating-campaign/membership/creating tier/deployed contract.png:1]: I think we can use this image for both event and membership section

**Comment by MoongateIntern** [comment]
For event tickets:
image


For memberships:
image

**Review by prabhanshjn — APPROVED** [review]
@MoongateIntern,
Please merge to main and update mintlify,
Thanks

**Pull request merged**